### ISSUE TITLE
Touch interrupt notworking

### ISSUE THREAD

**Issue by feaver12** [andrewfernie/CADDeck#2]
I have managed to get the CADDeck semi-working, however it seems that the TOUCH interrupt is not working... The UI is very laggy, as if touch only works in the loop, and the webserver does not load sometimes.  i get and error in serial console - "Guru Meditation Error: Core 1 panic'ed (InstrFetchProhibited). Exception was unhandled."

i have no idea what that error is referencing or where to start looking to fix these issues. Please help if you can see this. The caddeck commands are working with F360. Its mainly just the interrupt for the TFT that is not working..

these errors also just occured - [ 87967][E][vfs_api.cpp:104] open(): /littlefs/info does not exist, no permits for creation
[ 87972][E][vfs_api.cpp:104] open(): /littlefs/info.gz does not exist, no permits for creation
[ 87979][E][vfs_api.cpp:104] open(): /littlefs/info/index.htm does not exist, no permits for creation      
[ 87986][E][vfs_api.cpp:104] open(): /littlefs/info/index.htm.gz does not exist, no permits for creation

but i have already implemented the fix i found in littlefs github.... unless jm missing something

**Comment by andrewfernie** [comment]
Which hardware configuration are you using? Is it the "Current "AFUDirk" hardware" version  shown in the photo at the welcome page for this repository? And are you using the "main" branch?
1. The touch interrupt is used only to wake CADDeck from sleep mode when you touch the screen. It is not used for the general touch panel status to know which button has been pressed. The touch panel status is polled in loop() and if any data is found it is processed.
2. Lets separate the web configurator issue from the normal operation controlling F360.  Does it work properly then, or is it laggy in normal operation? Is the "Guru Meditation Error" connected with the web configurator, or does it happen in normal operation also?
3. As for web configurator...  The more data I added to the web screens the slower and less reliable it seemed to get. Most of the time I ended up editing the json files manually on my PC then uploading the updated files with the PlatformIO "Upload Filesystem Image" command in  VSCode. I wish I could figure out how to get this working but have not been successful. I know that editing codes in json files in a text editor is a pain so have worked on a PC app "CADDeckConfigurator" (https://github.com/andrewfernie/CADDeckConfigurator). It still requires the json files to be uploaded in VSCode, but at least all of the editing is done with a reasonable GUI.  One hassle at the moment is that it is configured for my CADDeck version that adds a small round LCD to the top of the knob. If you don't have that then I can try making the configurator adapt to the earlier version of CADDeck.

**Comment by feaver12** [comment]
Yes i was using the main branch in the mentioned repository, but i did notice the "knob" branch was still marked as active with the most recent updates, so i tried that and there was no difference.  Im not too worried about the webserver, as the commands from the touch screen seem to work so far, and i dont mind editing the files manually... just a little confused on what values to input in the json files to call specific keystrokes/shortcut keys.... i will have a look at the configurator tonight. 

As far as the guru mediation error and laggy ui goes - yes it is occurring during normal operation/idle. The esp will obviously reboot when this occurs which is a slightly annoying problem because of the boot time etc. 
There is one big thing to note though - i have not connected any buttons or joystick etc to the wt32-sc01 yet, as i am waiting for the buttons and rest of the parts needed to arrive from china - and am about to order pcb from jlcpcb later today... so im guessing that could be the main cause of the mediation error...... i will also order one of those small touchscreen for the knob - thats a really cool add on!.  
question - the gerber files in this repository are up to date? Or are there some more recent files somewhere? 

One more question related to the task handling of the UI - 

Is there any reason in particular that you are not using interrupts for the tft? The lack of response from the tft is a little offputting,  its usable, but i feel like it can be much better - having to hold the button until the loop comes back around to that section of the program is a little slow... i have made a few small programs before that utilize interrupts for this very reason, but nothing that has anywhere near as much happening as there is in this project..... if you like i could send you the code i used as a reference.... up to u.... 

By the way -

Have you thought about swapping the ui handling over to LVGL? The library has "built in" task handlers which make all of this sort of thing a breeze to implement... if not, im sure it would not be too hard to move the logic out of the loop and use an interrupt instead? I could have a look through some of my old projects later, and see if i can try to implement something to make the ui more responsive maybe.... its a little bit daunting because of the size/layout of this project is a bit different to what i have done in the past but i can try lol. Unless there are specific reasons why you have this part of the code operating in the loop? Or not using LVGL? 

Anyway - i really like the project! Keep up the good work! I cant wait until the stuff i have ordered turns up so i can try and get this completed
